README.md

```diff
@@ -69,4 +69,4 @@ This section has moved here: [https://facebook.github.io/create-react-app/docs/d
 
 This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
 
-anything you wanna add... add it in the 'features' branch i created, Ok?
```

Comment:
This line to tell you not to approch this branch... and anything you want to add... you can add it in the features branch